README.md

```diff
@@ -308,6 +308,20 @@ let table = TableContext<WorkItemInfo>(client, tableName = "workItems", metricsC
 If `metricsCollector` is supplied, the requests will set `ReturnConsumedCapacity` to `ReturnConsumedCapacity.INDEX` 
 and the `RequestMetrics` parameter will contain a list of `ConsumedCapacity` objects returned from the DynamoDB operations.
 
+## Read consistency
+
+DynamoDB follows an [eventually consistent model](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadConsistency.html) by default.
+As a consequence, data returned from a read operation might not reflect the changes of the most recently performed write operation if they are made in quick succession.
+To circumvent this limitation and enforce strongly consistent reads, DynamoDB provides a `ConsistentRead` parameter for read operations.
```

Comment:
It may not be house style, but I'd like to see a mention of the increased RU and latency implications and/or a link to a relevant section of the docs for the underlying API so people can look that up. (I know it's obvious if you come in from having done research, but this lib does make it extremely easy to roll out stuff very quickly)

Reply:
Yep, I'll add something